src/main/java/frc/trigon/robot/commands/VisualizeNoteShootingCommand.java

```diff
@@ -77,6 +83,24 @@ private void configureStartingStats() {
         initialZVelocity = startingPitch.getSin() * startingTangentialVelocity;
     }
 
+    private boolean noteCrossedFieldWithBorder(double noteY) {
+        return noteY > FieldConstants.FIELD_WIDTH_METERS && !crossedFieldWidthBorder;
```

Comment:
Width or with? 
I really don't understand this naming. What's the point of `border`?